roles/zabbix_agent/tasks/install-Windows.yml

```diff
@@ -70,7 +70,7 @@
         url_password: "{{ zabbix_download_pass | default(omit) }}"
         force: false
         follow_redirects: all
-        proxy_url: "{{ zabbix_https_proxy | default(None) | default(omit) }}"
+        proxy_url: "{{ zabbix_https_proxy | default(omit) }}"
```

Comment:
Should the be omit or '' ?

Reply:
It's a parameter to a module, so omit works (or should work) as intended here.